developer-docs/server-installation/plugins.md

```diff
@@ -0,0 +1,70 @@
+
+### Prerequisites Before Deployment
+
+**Create Containers**
+
+Create containers in Azure blob. Use the following commands to create initial placeholder containers:
+
+```
+az storage blob upload-batch --destination container_name/collection-editor --source some_folder --account-name storage_account_name --account-key account_key
+
+az storage blob upload-batch --destination container_name/generic-editor --source some_folder --account-name storage_account_name --account-key account_key
+
+az storage blob upload-batch --destination container_name/content-editor --source some_folder --account-name storage_account_name --account-key account_key
+
+az storage blob upload-batch --destination container_name/v3/preview --source some_folder --account-name storage_account_name --account-key account_key
+```
+
+> **Note:**
+1. The commands only create the required directories in Azure blob. The deployment script always tries to delete the folders before deploying new contents. Hence, if these folders are not available during the first run, the deployment script fails.
+2. You can upload any content in the blob for the first run. Azure does not allow you to create emtpy folders. Hence, add a file in a folder named *dummy*. The commands mentioned create the required directories.
+3. The **container_name** container must be publicly accessible.
+
+
+**Upload Base Plugin**
+
+Upload the intial set of plugins (base plugins) to your publicly accessible Azure container. 
+> **Note:** The name of the Azure container must be same as the **container_name** mentioned in the section on creating containers.
+
+You can download the intial Plugins from [here](https://sunbirdpublic.blob.core.windows.net/installation/content-plugins.zip)
+
+Unzip the contents and the directory **content-plugins** is available. Run the following command from the directory where the zip was extracted.
+
+```
+az storage blob upload-batch --destination container_name/content-plugins --source content-plugins --account-name account_name --account-key account_key
+```
+
+## Deploy
+
+Switch to `Deploy/<env>/DataPipeline` and run the jobs in the following sequence:
+
+1.ContentPlugins  
+2.ContentPlayer  
+3.CollectionEditor  
+4.ContentEditor  
+5.GenericEditor  
+
+After the plugins are deployed, you get URL's for the editors zip file. Sample URLs will be similar to the following:  
+
+**https://{{azure_account_name}}.blob.core.windows.net/{{plugin_container_name}}/artefacts/editor/generic-editor-iframe-2.4.0.zip**
+
+**https://{{azure_account_name}}.blob.core.windows.net/{{plugin_container_name}}/artefacts/editor/collection-editor-iframe-2.4.0.zip**
+
+**https://{{azure_account_name}}.blob.core.windows.net/{{plugin_container_name}}/artefacts/editor/content-editor-iframe-2.4.0.zip**
+
+You will need to use these 3 URL's when building Player.
```

Comment:
@keshavprasadms could you please clarify if this statement is right or the sequence for deployment at lines 55- 60 is correct? If the statement is correct, then the Player should be deployed after the Editors, if the sequence is correct, then this statement is not correct.

Reply:
@Sunitashelar The statement is correct. After all the above only, the user can build Player service which is mentioned in this page - https://github.com/keshavprasadms/sunbird.org-docs/blob/2.4.0/developer-docs/server-installation/core-services.md